src/search_algo.hpp

```diff
@@ -422,15 +422,44 @@ search_impl(LocalDataHolder<TGlobalHolder> & lH, TSeed && seed)
     }
     else if constexpr (TGlobalHolder::c_dbIndexType == DbIndexType::BI_FM_INDEX)
     {
-        fmindex_collection::search_pseudo::search</*editdistance=*/false>(
-            lH.gH.indexFile.index,
-            seed | seqan3::views::to_rank | fmindex_collection::add_sentinel,
-            lH.searchScheme,
-            [&](auto cursor, size_t /*errors*/)
-            {
-                lH.cursor_buffer.push_back(cursor);
-            }
-        );
+        if (lH.options.maxSeedDist == 0) {
```

Comment:
```suggestion
        if (lH.options.maxSeedDist == 0)
        {
```

Can you fix the style?

I am preparing to add clang-format soon